.goreleaser.yml

```
@@ -0,0 +1,87 @@
+# .goreleaser.yml proposed options for Romie
+builds:
+  # You can have multiple builds defined as a yaml list
+  - main: main.go
+    id: "romie"
+    dir: .
+    binary: romie
+
+    goos:
+      - linux
+      - darwin
+      - windows
+      - freebsd
+      - openbsd
+    goarch:
+      - 386
+      - amd64
+      - arm
+      - arm64
+    ldflags:
+     - -s -w -X main.build={{.Version}}
```

We need to be more specific for debugging purposes, so we need to know the `commit` and the `date` as well. On top of it, we need to be aware if the binary in use if made using `goreleaser` itself or the developer run `make build`.

I found this snippet. Please test it and give feedback if works as expected:

```suggestion
     - -s -w -X main.version={{.Version}} -X main.commit={{.Commit}} -X main.date={{ .CommitDate }} -X main.builtBy=goreleaser
```

---

I didn't see any difference in version number. Probably needs more investigations. For now I think it works as requested

---

It's normal that you don't see any difference now. These will be used later when we will implement `version` flag. e.g. `romie version`. It will use the information from the build ;)